app/forms/idv/validate_document_capture_session_form.rb

```diff
@@ -0,0 +1,51 @@
+module Idv
+  class ValidateDocumentCaptureSessionForm
```

Comment:
Is `Validate` a redundant prefix in this context? (Aren't all forms validating something?)

Reply:
I could go either way, since the form isn't doing much with the contents of the session besides checking they expire?
But I'm a big fan of shorter names, so I updated in 9f00d7b841d088efe8aa5579cae3e51ec63c4b9e